lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -78,7 +79,7 @@ spec = do
             Stdout mnemonics2 <- generateMnemonicsViaCLI ["--size", size]
             let passphrase = "Secure passphrase"
 
-            (c, out, err) <- createWalletViaCLI [walletName] mnemonics mnemonics2 passphrase
+            (c, out, err) <- createWalletViaCLI [walletName] mnemonics mnemonics2 passphrase (Just "1337")
```

Comment:
Also here! :) https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters